lab build error for 2.1.4 and 2.2.0a0

## Description

lab build error

## Reproduce

install any extensions and run
```
jupyter lab build
```

## Expected behavior

build should be completed successfully

## Context

<!--Complete the following for context, and add any other relevant context-->

- Operating System and version: Ubuntu 18.04
- Browser and version:  chrome for macos 83
- JupyterLab version: 2.1.4(conda) & 2.2.0a0(pip) (tried both )

<details><summary>Troubleshoot Output</summary>
<pre>
$PATH:
	/usr/local/cuda-10.2/bin
	/home/wmind/repo/fastText
	/home/wmind/repo/snap/examples/node2vec
	/home/wmind/anaconda3/bin
	/usr/local/sbin
	/usr/local/bin
	/usr/sbin
	/usr/bin
	/sbin
	/bin
	/usr/games
	/usr/local/games
	/snap/bin

sys.path:
	/home/wmind/anaconda3/bin
	/home/wmind/anaconda3/lib/python38.zip
	/home/wmind/anaconda3/lib/python3.8
	/home/wmind/anaconda3/lib/python3.8/lib-dynload
	/home/wmind/anaconda3/lib/python3.8/site-packages

sys.executable:
	/home/wmind/anaconda3/bin/python

sys.version:
	3.8.2 (default, Mar 26 2020, 15:53:00) 
	[GCC 7.3.0]

platform.platform():
	Linux-5.3.0-59-generic-x86_64-with-glibc2.10

which -a jupyter:
	/home/wmind/anaconda3/bin/jupyter

pip list:
	Package                            Version
	---------------------------------- -------------------
	absl-py                            0.9.0
	alabaster                          0.7.12
	anaconda-client                    1.7.2
	anaconda-project                   0.8.3
	argh                               0.26.2
	asn1crypto                         1.3.0
	astroid                            2.3.3
	astropy                            4.0.1.post1
	astunparse                         1.6.3
	atomicwrites                       1.4.0
	attrs                              19.3.0
	autopep8                           1.4.4
	Babel                              2.8.0
	backcall                           0.2.0
	backports.shutil-get-terminal-size 1.0.0
	beautifulsoup4                     4.9.1
	bitarray                           1.2.2
	bkcharts                           0.2
	bleach                             3.1.5
	blinker                            1.4
	bokeh                              2.1.0
	boto                               2.49.0
	Bottleneck                         1.3.2
	brotlipy                           0.7.0
	cachetools                         4.1.0
	certifi                            2020.4.5.2
	cffi                               1.14.0
	chardet                            3.0.4
	click                              7.1.2
	cloudpickle                        1.4.1
	clyent                             1.2.2
	colorama                           0.4.3
	conda                              4.8.3
	conda-package-handling             1.7.0
	contextlib2                        0.6.0.post1
	cryptography                       2.9.2
	cycler                             0.10.0
	Cython                             0.29.20
	cytoolz                            0.10.1
	dask                               2.18.1
	decorator                          4.4.2
	defusedxml                         0.6.0
	diff-match-patch                   20181111
	dill                               0.3.1.1
	distributed                        2.18.0
	dnspython                          1.16.0
	docutils                           0.16
	entrypoints                        0.3
	et-xmlfile                         1.0.1
	fastcache                          1.1.0
	filelock                           3.0.12
	flake8                             3.7.9
	Flask                              1.1.2
	fsspec                             0.7.4
	future                             0.18.2
	gast                               0.3.3
	gevent                             1.3.7
	glob2                              0.7
	gmpy2                              2.0.8
	google-api-core                    1.17.0
	google-auth                        1.14.1
	google-auth-oauthlib               0.4.1
	google-cloud-bigquery              1.24.0
	google-cloud-core                  1.3.0
	google-pasta                       0.2.0
	google-resumable-media             0.5.0
	googleapis-common-protos           1.51.0
	greenlet                           0.4.16
	grpcio                             1.27.2
	grpcio-tools                       1.27.2
	h5py                               2.10.0
	HeapDict                           1.0.1
	html5lib                           1.0.1
	hypothesis                         5.16.1
	idna                               2.9
	imageio                            2.8.0
	imagesize                          1.2.0
	importlib-metadata                 1.6.1
	intervaltree                       3.0.2
	ipykernel                          5.3.0
	ipython                            7.15.0
	ipython-genutils                   0.2.0
	ipywidgets                         7.5.1
	isort                              4.3.21
	itsdangerous                       1.1.0
	jdcal                              1.4.1
	jedi                               0.15.2
	jeepney                            0.4.3
	Jinja2                             2.11.2
	joblib                             0.15.1
	json5                              0.9.5
	jsonschema                         3.2.0
	jupyter                            1.0.0
	jupyter-client                     6.1.3
	jupyter-console                    6.1.0
	jupyter-core                       4.6.3
	jupyter-http-over-ws               0.0.8
	jupyterlab                         2.2.0a0
	jupyterlab-server                  1.1.5
	Keras                              2.3.1
	Keras-Applications                 1.0.8
	Keras-Preprocessing                1.1.2
	keyring                            21.1.1
	kiwisolver                         1.2.0
	lazy-object-proxy                  1.5.0
	libarchive-c                       2.9
	lightgbm                           3.0.0rc1
	llvmlite                           0.32.1
	locket                             0.2.0
	lxml                               4.5.1
	Markdown                           3.1.1
	MarkupSafe                         1.1.1
	matplotlib                         3.2.1
	mccabe                             0.6.1
	mistune                            0.8.4
	mkl-fft                            1.1.0
	mkl-random                         1.1.1
	mkl-service                        2.3.0
	mock                               4.0.2
	more-itertools                     8.4.0
	mpmath                             1.1.0
	msgpack                            1.0.0
	multipledispatch                   0.6.0
	nbconvert                          5.6.1
	nbformat                           5.0.7
	networkx                           2.4
	nltk                               3.4.5
	nose                               1.3.7
	notebook                           6.0.3
	numba                              0.49.1
	numexpr                            2.7.1
	numpy                              1.18.1
	numpydoc                           1.0.0
	oauthlib                           3.1.0
	olefile                            0.46
	opencv-python                      4.2.0.34
	openpyxl                           3.0.3
	opt-einsum                         3.1.0
	packaging                          20.4
	pandas                             1.0.4
	pandocfilters                      1.4.2
	parso                              0.5.2
	partd                              1.1.0
	path                               13.1.0
	pathlib2                           2.3.5
	pathtools                          0.1.2
	patsy                              0.5.1
	pep8                               1.7.1
	pexpect                            4.8.0
	pickleshare                        0.7.5
	Pillow                             7.1.2
	pip                                20.1.1
	pkginfo                            1.5.0.1
	plotly                             4.8.1
	pluggy                             0.13.1
	ply                                3.11
	prometheus-client                  0.8.0
	promise                            2.2.1
	prompt-toolkit                     3.0.5
	protobuf                           3.12.3
	psutil                             5.7.0
	ptyprocess                         0.6.0
	py                                 1.8.2
	pyasn1                             0.4.8
	pyasn1-modules                     0.2.7
	pycodestyle                        2.5.0
	pycosat                            0.6.3
	pycparser                          2.20
	pycurl                             7.43.0.5
	pydocstyle                         4.0.1
	pyflakes                           2.1.1
	Pygments                           2.6.1
	PyJWT                              1.7.1
	pylint                             2.4.4
	pymongo                            3.9.0
	pyodbc                             4.0.0-unsupported
	pyOpenSSL                          19.1.0
	pyparsing                          2.4.7
	pyrsistent                         0.16.0
	PySocks                            1.7.1
	pytest                             5.4.3
	pytest-arraydiff                   0.2
	pytest-astropy                     0.8.0
	pytest-astropy-header              0.1.2
	pytest-doctestplus                 0.7.0
	pytest-openfiles                   0.5.0
	pytest-remotedata                  0.3.2
	python-dateutil                    2.8.1
	python-jsonrpc-server              0.3.4
	python-language-server             0.31.10
	pytz                               2020.1
	PyWavelets                         1.1.1
	pyxdg                              0.26
	PyYAML                             5.3.1
	pyzmq                              18.1.1
	QDarkStyle                         2.8.1
	QtAwesome                          0.7.2
	qtconsole                          4.7.4
	QtPy                               1.9.0
	requests                           2.23.0
	requests-oauthlib                  1.3.0
	retrying                           1.3.3
	rope                               0.17.0
	rsa                                4.0
	Rtree                              0.9.4
	ruamel-yaml                        0.15.87
	scikit-image                       0.16.2
	scikit-learn                       0.22.1
	scipy                              1.4.1
	seaborn                            0.10.1
	SecretStorage                      3.1.2
	Send2Trash                         1.5.0
	setuptools                         47.3.0.post20200616
	simplegeneric                      0.8.1
	singledispatch                     3.4.0.3
	sip                                4.19.13
	six                                1.15.0
	snowballstemmer                    2.0.0
	sortedcollections                  1.2.1
	sortedcontainers                   2.2.2
	soupsieve                          2.0.1
	Sphinx                             3.0.4
	sphinxcontrib-applehelp            1.0.2
	sphinxcontrib-devhelp              1.0.2
	sphinxcontrib-htmlhelp             1.0.3
	sphinxcontrib-jsmath               1.0.1
	sphinxcontrib-qthelp               1.0.3
	sphinxcontrib-serializinghtml      1.1.4
	sphinxcontrib-websupport           1.2.1
	spyder                             4.1.3
	spyder-kernels                     1.9.1
	SQLAlchemy                         1.3.17
	statsmodels                        0.11.1
	sympy                              1.6
	tables                             3.6.1
	tblib                              1.6.0
	tensorboard                        2.2.2
	tensorboard-plugin-wit             1.6.0.post3
	tensorflow                         2.2.0
	tensorflow-datasets                3.1.0
	tensorflow-estimator               2.2.0
	tensorflow-hub                     0.8.0
	tensorflow-metadata                0.22.0
	tensorflow-probability             0.10.0
	termcolor                          1.1.0
	terminado                          0.8.3
	testpath                           0.4.4
	toolz                              0.10.0
	torch                              1.5.0
	torchvision                        0.6.0
	tornado                            6.0.4
	tqdm                               4.46.0
	traitlets                          4.3.3
	typing-extensions                  3.7.4.1
	ujson                              1.35
	unicodecsv                         0.14.1
	urllib3                            1.25.9
	watchdog                           0.10.2
	wcwidth                            0.2.4
	webencodings                       0.5.1
	Werkzeug                           1.0.1
	wheel                              0.34.2
	widgetsnbextension                 3.5.1
	wrapt                              1.12.1
	wurlitzer                          2.0.0
	xgboost                            1.1.0
	xlrd                               1.2.0
	XlsxWriter                         1.2.9
	xlwt                               1.3.0
	yapf                               0.28.0
	zict                               2.0.0
	zipp                               3.1.0

conda list:
	# packages in environment at /home/wmind/anaconda3:
	#
	# Name                    Version                   Build  Channel
	_anaconda_depends         2020.02                  py38_0  
	_libgcc_mutex             0.1                        main  
	absl-py                   0.9.0                    py38_0  
	alabaster                 0.7.12                     py_0  
	anaconda                  custom                   py38_1  
	anaconda-client           1.7.2                    py38_0  
	anaconda-project          0.8.4                      py_0  
	argh                      0.26.2                   py38_0  
	asn1crypto                1.3.0                    py38_0  
	astroid                   2.3.3                    py38_0  
	astropy                   4.0.1.post1      py38h7b6447c_1  
	astunparse                1.6.3                      py_0  
	atomicwrites              1.4.0                      py_0  
	attrs                     19.3.0                     py_0  
	autopep8                  1.4.4                      py_0  
	babel                     2.8.0                      py_0  
	backcall                  0.2.0                      py_0  
	backports                 1.0                        py_2  
	backports.shutil_get_terminal_size 1.0.0                    py38_2  
	beautifulsoup4            4.9.1                    py38_0  
	bitarray                  1.2.2            py38h7b6447c_0  
	bkcharts                  0.2                      py38_0  
	blas                      1.0                         mkl  
	bleach                    3.1.5                      py_0  
	blinker                   1.4                      py38_0  
	blosc                     1.16.3               hd408876_0  
	bokeh                     2.1.0                    py38_0  
	boto                      2.49.0                   py38_0  
	bottleneck                1.3.2            py38heb32a55_1  
	brotlipy                  0.7.0           py38h7b6447c_1000  
	bzip2                     1.0.8                h7b6447c_0  
	c-ares                    1.15.0            h7b6447c_1001  
	ca-certificates           2020.1.1                      0  
	cachetools                4.1.0                      py_1  
	cairo                     1.14.12              h8948797_3  
	certifi                   2020.4.5.2               py38_0  
	cffi                      1.14.0           py38h2e261b9_0  
	chardet                   3.0.4                 py38_1003  
	click                     7.1.2                      py_0  
	cloudpickle               1.4.1                      py_0  
	clyent                    1.2.2                    py38_1  
	colorama                  0.4.3                      py_0  
	conda                     4.8.3                    py38_0  
	conda-package-handling    1.6.1            py38h7b6447c_0  
	contextlib2               0.6.0.post1                py_0  
	cryptography              2.9.2            py38h1ba5d50_0  
	curl                      7.69.1               hbc83047_0  
	cycler                    0.10.0                   py38_0  
	cython                    0.29.20          py38he6710b0_0  
	cytoolz                   0.10.1           py38h7b6447c_0  
	dask                      2.18.1                     py_0  
	dask-core                 2.18.1                     py_0  
	dbus                      1.13.14              hb2f20db_0  
	decorator                 4.4.2                      py_0  
	defusedxml                0.6.0                      py_0  
	diff-match-patch          20181111                   py_0  
	dill                      0.3.1.1                  py38_1  
	distributed               2.18.0                   py38_0  
	dnspython                 1.16.0                   py38_0  
	docutils                  0.16                     py38_1  
	entrypoints               0.3                      py38_0  
	et_xmlfile                1.0.1                    py38_0  
	expat                     2.2.6                he6710b0_0  
	fastcache                 1.1.0            py38h7b6447c_0  
	filelock                  3.0.12                     py_0  
	flake8                    3.7.9                    py38_0  
	flask                     1.1.2                      py_0  
	fontconfig                2.13.0               h9420a91_0  
	freetype                  2.9.1                h8a8886c_1  
	fribidi                   1.0.5                h7b6447c_0  
	fsspec                    0.7.4                      py_0  
	future                    0.18.2                   py38_1  
	gast                      0.3.3                      py_0  
	get_terminal_size         1.0.0                haa9412d_0  
	gevent                    1.3.7            py38h7b6447c_1  
	glib                      2.63.1               h5a9c865_0  
	glob2                     0.7                        py_0  
	gmp                       6.1.2                h6c8ec71_1  
	gmpy2                     2.0.8            py38hd5f6e3b_3  
	google-api-core           1.17.0                   py38_0  
	google-auth               1.14.1                     py_0  
	google-auth-oauthlib      0.4.1                      py_2  
	google-cloud-bigquery     1.24.0                   pypi_0    pypi
	google-cloud-core         1.3.0                      py_0  
	google-pasta              0.2.0                      py_0  
	google-resumable-media    0.5.0                      py_1  
	googleapis-common-protos  1.51.0                   py38_2  
	graphite2                 1.3.13               h23475e2_0  
	greenlet                  0.4.16           py38h7b6447c_0  
	grpcio                    1.27.2           py38hf8bcb03_0  
	grpcio-tools              1.27.2                   pypi_0    pypi
	gst-plugins-base          1.14.0               hbbd80ab_1  
	gstreamer                 1.14.0               hb453b48_1  
	h5py                      2.10.0           py38h7918eee_0  
	harfbuzz                  1.8.8                hffaf4a1_0  
	hdf5                      1.10.4               hb1b8bf9_0  
	heapdict                  1.0.1                      py_0  
	html5lib                  1.0.1                      py_0  
	hypothesis                5.16.1                     py_0  
	icu                       58.2                 he6710b0_3  
	idna                      2.9                        py_1  
	imageio                   2.8.0                      py_0  
	imagesize                 1.2.0                      py_0  
	importlib-metadata        1.6.1                    py38_0  
	importlib_metadata        1.6.1                         0  
	intel-openmp              2020.1                      217  
	intervaltree              3.0.2                      py_0  
	ipykernel                 5.3.0            py38h5ca1d4c_0  
	ipython                   7.15.0                   py38_0  
	ipython_genutils          0.2.0                    py38_0  
	ipywidgets                7.5.1                      py_0  
	isort                     4.3.21                   py38_0  
	itsdangerous              1.1.0                      py_0  
	jbig                      2.1                  hdba287a_0  
	jdcal                     1.4.1                      py_0  
	jedi                      0.15.2                   py38_0  
	jeepney                   0.4.3                      py_0  
	jinja2                    2.11.2                     py_0  
	joblib                    0.15.1                     py_0  
	jpeg                      9b                   h024ee3a_2  
	json5                     0.9.5                      py_0  
	jsonschema                3.2.0                    py38_0  
	jupyter                   1.0.0                    py38_7  
	jupyter-http-over-ws      0.0.8                    pypi_0    pypi
	jupyter_client            6.1.3                      py_0  
	jupyter_console           6.1.0                      py_0  
	jupyter_core              4.6.3                    py38_0  
	jupyterlab                2.2.0a0                  pypi_0    pypi
	jupyterlab_server         1.1.5                      py_0  
	keras                     2.3.1                    pypi_0    pypi
	keras-applications        1.0.8                    pypi_0    pypi
	keras-preprocessing       1.1.2                    pypi_0    pypi
	keyring                   21.1.1                   py38_2  
	kiwisolver                1.2.0            py38hfd86e86_0  
	krb5                      1.17.1               h173b8e3_0  
	lazy-object-proxy         1.5.0            py38h7b6447c_0  
	ld_impl_linux-64          2.33.1               h53a641e_7  
	libarchive                3.3.3                h5d8350f_5  
	libcurl                   7.69.1               h20c2e04_0  
	libedit                   3.1.20181209         hc058e9b_0  
	libffi                    3.2.1                hd88cf55_4  
	libgcc-ng                 9.1.0                hdf63c60_0  
	libgfortran-ng            7.3.0                hdf63c60_0  
	libpng                    1.6.37               hbc83047_0  
	libprotobuf               3.12.3               hd408876_0  
	libsodium                 1.0.16               h1bed415_0  
	libspatialindex           1.9.3                he6710b0_0  
	libssh2                   1.9.0                h1ba5d50_1  
	libstdcxx-ng              9.1.0                hdf63c60_0  
	libtiff                   4.1.0                h2733197_0  
	libtool                   2.4.6                h7b6447c_5  
	libuuid                   1.0.3                h1bed415_2  
	libxcb                    1.13                 h1bed415_1  
	libxml2                   2.9.10               he19cac6_1  
	libxslt                   1.1.34               hc22bd24_0  
	lightgbm                  3.0.0rc1                 pypi_0    pypi
	llvmlite                  0.32.1           py38hd408876_0  
	locket                    0.2.0                    py38_1  
	lxml                      4.5.1            py38hefd8a0e_0  
	lz4-c                     1.8.1.2              h14c3975_0  
	lzo                       2.10                 h7b6447c_2  
	markdown                  3.1.1                    py38_0  
	markupsafe                1.1.1            py38h7b6447c_0  
	matplotlib                3.2.1                         0  
	matplotlib-base           3.2.1            py38hef1b27d_0  
	mccabe                    0.6.1                    py38_1  
	mistune                   0.8.4           py38h7b6447c_1000  
	mkl                       2020.1                      217  
	mkl-service               2.3.0            py38he904b0f_0  
	mkl_fft                   1.1.0            py38h23d657b_0  
	mkl_random                1.1.1            py38h0573a6f_0  
	mock                      4.0.2                      py_0  
	more-itertools            8.4.0                      py_0  
	mpc                       1.1.0                h10f8cd9_1  
	mpfr                      4.0.1                hdf1c602_3  
	mpmath                    1.1.0                    py38_0  
	msgpack-python            1.0.0            py38hfd86e86_1  
	multipledispatch          0.6.0                    py38_0  
	nbconvert                 5.6.1                    py38_0  
	nbformat                  5.0.7                      py_0  
	ncurses                   6.2                  he6710b0_1  
	networkx                  2.4                        py_0  
	nltk                      3.4.5                    py38_0  
	nose                      1.3.7                    py38_2  
	notebook                  6.0.3                    py38_0  
	numba                     0.49.1           py38h0573a6f_0  
	numexpr                   2.7.1            py38h423224d_0  
	numpy                     1.18.1           py38h4f9e942_0  
	numpy-base                1.18.1           py38hde5b4d6_1  
	numpydoc                  1.0.0                      py_0  
	oauthlib                  3.1.0                      py_0  
	olefile                   0.46                       py_0  
	opencv-python             4.2.0.34                 pypi_0    pypi
	openpyxl                  3.0.3                      py_0  
	openssl                   1.1.1g               h7b6447c_0  
	opt_einsum                3.1.0                      py_0  
	packaging                 20.4                       py_0  
	pandas                    1.0.4            py38h0573a6f_0  
	pandoc                    2.2.3.2                       0  
	pandocfilters             1.4.2                    py38_1  
	pango                     1.42.4               h049681c_0  
	parso                     0.5.2                      py_0  
	partd                     1.1.0                      py_0  
	patchelf                  0.11                 he6710b0_0  
	path                      13.1.0                   py38_0  
	path.py                   12.4.0                        0  
	pathlib2                  2.3.5                    py38_0  
	pathtools                 0.1.2                      py_1  
	patsy                     0.5.1                    py38_0  
	pcre                      8.43                 he6710b0_0  
	pep8                      1.7.1                    py38_0  
	pexpect                   4.8.0                    py38_0  
	pickleshare               0.7.5                 py38_1000  
	pillow                    7.1.2            py38hb39fc2d_0  
	pip                       20.1.1                   py38_1  
	pixman                    0.38.0               h7b6447c_0  
	pkginfo                   1.5.0.1                  py38_0  
	plotly                    4.8.1                      py_0  
	pluggy                    0.13.1                   py38_0  
	ply                       3.11                     py38_0  
	prometheus_client         0.8.0                      py_0  
	promise                   2.2.1                    py38_0  
	prompt-toolkit            3.0.5                      py_0  
	prompt_toolkit            3.0.5                         0  
	protobuf                  3.12.3           py38he6710b0_0  
	psutil                    5.7.0            py38h7b6447c_0  
	ptyprocess                0.6.0                    py38_0  
	py                        1.8.2                      py_0  
	pyasn1                    0.4.8                      py_0  
	pyasn1-modules            0.2.7                      py_0  
	pycodestyle               2.5.0                    py38_0  
	pycosat                   0.6.3            py38h7b6447c_1  
	pycparser                 2.20                       py_0  
	pycurl                    7.43.0.5         py38h1ba5d50_0  
	pydocstyle                4.0.1                      py_0  
	pyflakes                  2.1.1                    py38_0  
	pygments                  2.6.1                      py_0  
	pyjwt                     1.7.1                    py38_0  
	pylint                    2.4.4                    py38_0  
	pymongo                   3.9.0            py38he6710b0_0  
	pyodbc                    4.0.30           py38he6710b0_0  
	pyopenssl                 19.1.0                   py38_0  
	pyparsing                 2.4.7                      py_0  
	pyqt                      5.9.2            py38h05f1152_4  
	pyrsistent                0.16.0           py38h7b6447c_0  
	pysocks                   1.7.1                    py38_0  
	pytables                  3.6.1            py38h9fd0a39_0  
	pytest                    5.4.3                    py38_0  
	pytest-arraydiff          0.2              py38h39e3cac_0  
	pytest-astropy            0.8.0                      py_0  
	pytest-astropy-header     0.1.2                      py_0  
	pytest-doctestplus        0.7.0                      py_0  
	pytest-openfiles          0.5.0                      py_0  
	pytest-remotedata         0.3.2                    py38_0  
	python                    3.8.2                hcf32534_0  
	python-dateutil           2.8.1                      py_0  
	python-jsonrpc-server     0.3.4                      py_0  
	python-language-server    0.31.10                  py38_0  
	python-libarchive-c       2.9                        py_0  
	pytz                      2020.1                     py_0  
	pywavelets                1.1.1            py38h7b6447c_0  
	pyxdg                     0.26                       py_0  
	pyyaml                    5.3.1            py38h7b6447c_0  
	pyzmq                     18.1.1           py38he6710b0_0  
	qdarkstyle                2.8.1                      py_0  
	qt                        5.9.7                h5867ecd_1  
	qtawesome                 0.7.2                      py_0  
	qtconsole                 4.7.4                      py_0  
	qtpy                      1.9.0                      py_0  
	readline                  8.0                  h7b6447c_0  
	requests                  2.23.0                   py38_0  
	requests-oauthlib         1.3.0                      py_0  
	retrying                  1.3.3                      py_2  
	ripgrep                   11.0.2               he32d670_0  
	rope                      0.17.0                     py_0  
	rsa                       4.0                        py_0  
	rtree                     0.9.4                    py38_1  
	ruamel_yaml               0.15.87          py38h7b6447c_0  
	scikit-image              0.16.2           py38h0573a6f_0  
	scikit-learn              0.22.1           py38hd81dba3_0  
	scipy                     1.4.1            py38h0b6359f_0  
	seaborn                   0.10.1                     py_0  
	secretstorage             3.1.2                    py38_0  
	send2trash                1.5.0                    py38_0  
	setuptools                47.3.0                   py38_0  
	simplegeneric             0.8.1                    py38_2  
	singledispatch            3.4.0.3                  py38_0  
	sip                       4.19.13          py38he6710b0_0  
	six                       1.15.0                     py_0  
	snappy                    1.1.7                hbae5bb6_3  
	snowballstemmer           2.0.0                      py_0  
	sortedcollections         1.2.1                      py_0  
	sortedcontainers          2.2.2                      py_0  
	soupsieve                 2.0.1                      py_0  
	sphinx                    3.0.4                      py_0  
	sphinxcontrib             1.0                      py38_1  
	sphinxcontrib-applehelp   1.0.2                      py_0  
	sphinxcontrib-devhelp     1.0.2                      py_0  
	sphinxcontrib-htmlhelp    1.0.3                      py_0  
	sphinxcontrib-jsmath      1.0.1                      py_0  
	sphinxcontrib-qthelp      1.0.3                      py_0  
	sphinxcontrib-serializinghtml 1.1.4                      py_0  
	sphinxcontrib-websupport  1.2.1                      py_0  
	spyder                    4.1.3                    py38_0  
	spyder-kernels            1.9.1                    py38_0  
	sqlalchemy                1.3.17           py38h7b6447c_0  
	sqlite                    3.31.1               h62c20be_1  
	statsmodels               0.11.1           py38h7b6447c_0  
	sympy                     1.6                      py38_0  
	tbb                       2020.0               hfd86e86_0  
	tblib                     1.6.0                      py_0  
	tensorboard               2.2.2                    pypi_0    pypi
	tensorboard-plugin-wit    1.6.0.post3              pypi_0    pypi
	tensorflow                2.2.0                    pypi_0    pypi
	tensorflow-datasets       3.1.0                    pypi_0    pypi
	tensorflow-estimator      2.2.0                    pypi_0    pypi
	tensorflow-hub            0.8.0                    pypi_0    pypi
	tensorflow-metadata       0.22.0                   pypi_0    pypi
	tensorflow-probability    0.10.0                   pypi_0    pypi
	termcolor                 1.1.0                    py38_1  
	terminado                 0.8.3                    py38_0  
	testpath                  0.4.4                      py_0  
	tk                        8.6.8                hbc83047_0  
	toolz                     0.10.0                     py_0  
	torch                     1.5.0                    pypi_0    pypi
	torchvision               0.6.0                    pypi_0    pypi
	tornado                   6.0.4            py38h7b6447c_1  
	tqdm                      4.46.0                     py_0  
	traitlets                 4.3.3                    py38_0  
	typing_extensions         3.7.4.1                  py38_0  
	ujson                     1.35             py38h7b6447c_0  
	unicodecsv                0.14.1                   py38_0  
	unixodbc                  2.3.7                h14c3975_0  
	urllib3                   1.25.9                     py_0  
	watchdog                  0.10.2                   py38_0  
	wcwidth                   0.2.4                      py_0  
	webencodings              0.5.1                    py38_1  
	werkzeug                  1.0.1                      py_0  
	wheel                     0.34.2                   py38_0  
	widgetsnbextension        3.5.1                    py38_0  
	wrapt                     1.12.1           py38h7b6447c_1  
	wurlitzer                 2.0.0                    py38_0  
	xgboost                   1.1.0                    pypi_0    pypi
	xlrd                      1.2.0                      py_0  
	xlsxwriter                1.2.9                      py_0  
	xlwt                      1.3.0                    py38_0  
	xz                        5.2.5                h7b6447c_0  
	yaml                      0.1.7                had09818_2  
	yapf                      0.28.0                     py_0  
	zeromq                    4.3.1                he6710b0_3  
	zict                      2.0.0                      py_0  
	zipp                      3.1.0                      py_0  
	zlib                      1.2.11               h7b6447c_3  
	zstd                      1.3.7                h0b5b093_0
</pre>
</details>

<details><summary>Command Line Output</summary>
<pre>
[D 10:38:16.396 LabApp] Searching ['/mnt/sda1/newrec/data', '/home/wmind/.jupyter', '/home/wmind/anaconda3/etc/jupyter', '/usr/local/etc/jupyter', '/etc/jupyter'] for config files
[D 10:38:16.396 LabApp] Looking for jupyter_config in /etc/jupyter
[D 10:38:16.397 LabApp] Looking for jupyter_config in /usr/local/etc/jupyter
[D 10:38:16.397 LabApp] Looking for jupyter_config in /home/wmind/anaconda3/etc/jupyter
[D 10:38:16.397 LabApp] Looking for jupyter_config in /home/wmind/.jupyter
[D 10:38:16.397 LabApp] Looking for jupyter_config in /mnt/sda1/newrec/data
[D 10:38:16.398 LabApp] Looking for jupyter_notebook_config in /etc/jupyter
[D 10:38:16.399 LabApp] Looking for jupyter_notebook_config in /usr/local/etc/jupyter
[D 10:38:16.399 LabApp] Looking for jupyter_notebook_config in /home/wmind/anaconda3/etc/jupyter
[D 10:38:16.399 LabApp] Looking for jupyter_notebook_config in /home/wmind/.jupyter
[D 10:38:16.399 LabApp] Loaded config file: /home/wmind/.jupyter/jupyter_notebook_config.json
[D 10:38:16.399 LabApp] Looking for jupyter_notebook_config in /mnt/sda1/newrec/data
[D 10:38:16.400 LabApp] Raising open file limit: soft 1024->4096; hard 1048576->1048576
[D 10:38:16.404 LabApp] Paths used for configuration of jupyter_notebook_config: 
    	/etc/jupyter/jupyter_notebook_config.json
[D 10:38:16.404 LabApp] Paths used for configuration of jupyter_notebook_config: 
    	/usr/local/etc/jupyter/jupyter_notebook_config.json
[D 10:38:16.405 LabApp] Paths used for configuration of jupyter_notebook_config: 
    	/home/wmind/anaconda3/etc/jupyter/jupyter_notebook_config.d/jupyterlab.json
    	/home/wmind/anaconda3/etc/jupyter/jupyter_notebook_config.json
[D 10:38:16.405 LabApp] Paths used for configuration of jupyter_notebook_config: 
    	/home/wmind/.jupyter/jupyter_notebook_config.json
jupyter_http_over_ws extension initialized. Listening on /http_over_websocket
[I 10:38:17.000 LabApp] JupyterLab extension loaded from /home/wmind/anaconda3/lib/python3.8/site-packages/jupyterlab
[I 10:38:17.000 LabApp] JupyterLab application directory is /home/wmind/anaconda3/share/jupyter/lab
[I 10:38:17.006 LabApp] Serving notebooks from local directory: /mnt/sda1/newrec/data
[I 10:38:17.006 LabApp] The Jupyter Notebook is running at:
[I 10:38:17.006 LabApp] http://localhost:8888/?token=55a807dd1a8414e0d782706a7fe4f2acf324a4176a89d620
[I 10:38:17.006 LabApp]  or http://127.0.0.1:8888/?token=55a807dd1a8414e0d782706a7fe4f2acf324a4176a89d620
[I 10:38:17.006 LabApp] Use Control-C to stop this server and shut down all kernels (twice to skip confirmation).
[W 10:38:17.012 LabApp] No web browser found: could not locate runnable browser.
[C 10:38:17.012 LabApp] 
    
    To access the notebook, open this file in a browser:
        file:///home/wmind/.local/share/jupyter/runtime/nbserver-5381-open.html
    Or copy and paste one of these URLs:
        http://localhost:8888/?token=55a807dd1a8414e0d782706a7fe4f2acf324a4176a89d620
     or http://127.0.0.1:8888/?token=55a807dd1a8414e0d782706a7fe4f2acf324a4176a89d620
</pre>
</details>

<details><summary>Browser Output</summary>
<pre>
lab:1 A cookie associated with a cross-site resource at http://github.com/ was set without the `SameSite` attribute. A future release of Chrome will only deliver cookies with cross-site requests if they are set with `SameSite=None` and `Secure`. You can review cookies in developer tools under Application>Storage>Cookies and see more details at https://www.chromestatus.com/feature/5088147346030592 and https://www.chromestatus.com/feature/5633521622188032.
lab:1 A cookie associated with a cross-site resource at https://github.com/ was set without the `SameSite` attribute. A future release of Chrome will only deliver cookies with cross-site requests if they are set with `SameSite=None` and `Secure`. You can review cookies in developer tools under Application>Storage>Cookies and see more details at https://www.chromestatus.com/feature/5088147346030592 and https://www.chromestatus.com/feature/5633521622188032.
vendors~main.31198801f9db66b5f661.js:2 POST http://10.102.40.94:8888/lab/api/build?1592444392604 500 (Internal Server Error)
n @ vendors~main.31198801f9db66b5f661.js:2
n @ vendors~main.31198801f9db66b5f661.js:2
build @ vendors~main.31198801f9db66b5f661.js:2
c @ vendors~main.31198801f9db66b5f661.js:2
(anonymous) @ vendors~main.31198801f9db66b5f661.js:2
vendors~main.31198801f9db66b5f661.js:2 Showing error: {message: {…}}
d @ vendors~main.31198801f9db66b5f661.js:2
(anonymous) @ vendors~main.31198801f9db66b5f661.js:2
Promise.catch (async)
c @ vendors~main.31198801f9db66b5f661.js:2
(anonymous) @ vendors~main.31198801f9db66b5f661.js:2
</pre>
</details>

```
[LabBuildApp] Building in /home/wmind/anaconda3/share/jupyter/lab
[LabBuildApp] Node v12.18.1

[LabBuildApp] Yarn configuration loaded.
[LabBuildApp] Building jupyterlab assets (build:prod:minimize)
[LabBuildApp] > node /home/wmind/anaconda3/lib/python3.8/site-packages/jupyterlab/staging/yarn.js install --non-interactive
[LabBuildApp] yarn install v1.21.1
[1/5] Validating package.json...
[2/5] Resolving packages...
warning plotlywidget > plotly.js > regl-splom > left-pad@1.3.0: use String.prototype.padStart()
warning plotlywidget > plotly.js > point-cluster > bubleify > buble > os-homedir@2.0.0: This is not needed anymore. Use `require('os').homedir()` instead.
[3/5] Fetching packages...
info fsevents@2.1.3: The platform "linux" is incompatible with this module.
info "fsevents@2.1.3" is an optional dependency and failed compatibility check. Excluding it from installation.
info fsevents@1.2.13: The platform "linux" is incompatible with this module.
info "fsevents@1.2.13" is an optional dependency and failed compatibility check. Excluding it from installation.
[4/5] Linking dependencies...
warning " > @lumino/coreutils@1.5.0" has unmet peer dependency "crypto@1.0.1".
warning " > @jupyterlab/git@0.20.0" has unmet peer dependency "codemirror@^5.0.0".
warning "@jupyterlab/git > nbdime@6.0.0" has unmet peer dependency "codemirror@^5.0.0".
[5/5] Building fresh packages...
success Saved lockfile.
Done in 24.51s.

[LabBuildApp] > node /home/wmind/anaconda3/lib/python3.8/site-packages/jupyterlab/staging/yarn.js yarn-deduplicate -s fewer --fail
[LabBuildApp] yarn run v1.21.1
$ /home/wmind/anaconda3/share/jupyter/lab/staging/node_modules/.bin/yarn-deduplicate -s fewer --fail
error Command failed with exit code 1.
info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.

[LabBuildApp] > node /home/wmind/anaconda3/lib/python3.8/site-packages/jupyterlab/staging/yarn.js
[LabBuildApp] yarn install v1.21.1
[1/5] Validating package.json...
[2/5] Resolving packages...
[3/5] Fetching packages...
info fsevents@2.1.3: The platform "linux" is incompatible with this module.
info "fsevents@2.1.3" is an optional dependency and failed compatibility check. Excluding it from installation.
info fsevents@1.2.13: The platform "linux" is incompatible with this module.
info "fsevents@1.2.13" is an optional dependency and failed compatibility check. Excluding it from installation.
[4/5] Linking dependencies...
warning " > @lumino/coreutils@1.5.0" has unmet peer dependency "crypto@1.0.1".
warning " > @jupyterlab/git@0.20.0" has unmet peer dependency "codemirror@^5.0.0".
warning "@jupyterlab/git > nbdime@6.0.0" has unmet peer dependency "codemirror@^5.0.0".
[5/5] Building fresh packages...
Done in 5.25s.

[LabBuildApp] > node /home/wmind/anaconda3/lib/python3.8/site-packages/jupyterlab/staging/yarn.js run build:prod:minimize
[LabBuildApp] yarn run v1.21.1
$ ensure-max-old-space webpack --config webpack.prod.minimize.config.js
ModuleNotFoundError: Module not found: Error: Can't resolve 'worker-loader' in '/home/wmind/anaconda3/share/jupyter/lab/staging/node_modules/pdfjs-dist'
    at /home/wmind/anaconda3/share/jupyter/lab/staging/node_modules/webpack/lib/Compilation.js:925:10
    at /home/wmind/anaconda3/share/jupyter/lab/staging/node_modules/webpack/lib/NormalModuleFactory.js:401:22
    at /home/wmind/anaconda3/share/jupyter/lab/staging/node_modules/webpack/lib/NormalModuleFactory.js:130:21
    at /home/wmind/anaconda3/share/jupyter/lab/staging/node_modules/webpack/lib/NormalModuleFactory.js:224:22
    at /home/wmind/anaconda3/share/jupyter/lab/staging/node_modules/neo-async/async.js:2830:7
    at /home/wmind/anaconda3/share/jupyter/lab/staging/node_modules/neo-async/async.js:6877:13
    at /home/wmind/anaconda3/share/jupyter/lab/staging/node_modules/neo-async/async.js:2830:7
    at done (/home/wmind/anaconda3/share/jupyter/lab/staging/node_modules/neo-async/async.js:2925:13)
    at /home/wmind/anaconda3/share/jupyter/lab/staging/node_modules/webpack/lib/NormalModuleFactory.js:449:23
    at /home/wmind/anaconda3/share/jupyter/lab/staging/node_modules/enhanced-resolve/lib/Resolver.js:184:12
    at /home/wmind/anaconda3/share/jupyter/lab/staging/node_modules/enhanced-resolve/lib/Resolver.js:238:5
    at eval (eval at create (/home/wmind/anaconda3/share/jupyter/lab/staging/node_modules/tapable/lib/HookCodeFactory.js:33:10), <anonymous>:13:1)
    at /home/wmind/anaconda3/share/jupyter/lab/staging/node_modules/enhanced-resolve/lib/UnsafeCachePlugin.js:37:5
    at /home/wmind/anaconda3/share/jupyter/lab/staging/node_modules/enhanced-resolve/lib/Resolver.js:238:5
    at eval (eval at create (/home/wmind/anaconda3/share/jupyter/lab/staging/node_modules/tapable/lib/HookCodeFactory.js:33:10), <anonymous>:13:1)
    at /home/wmind/anaconda3/share/jupyter/lab/staging/node_modules/enhanced-resolve/lib/Resolver.js:238:5
    at eval (eval at create (/home/wmind/anaconda3/share/jupyter/lab/staging/node_modules/tapable/lib/HookCodeFactory.js:33:10), <anonymous>:25:1)
    at /home/wmind/anaconda3/share/jupyter/lab/staging/node_modules/enhanced-resolve/lib/DescriptionFilePlugin.js:42:38
    at /home/wmind/anaconda3/share/jupyter/lab/staging/node_modules/enhanced-resolve/lib/Resolver.js:238:5
    at eval (eval at create (/home/wmind/anaconda3/share/jupyter/lab/staging/node_modules/tapable/lib/HookCodeFactory.js:33:10), <anonymous>:25:1)
    at /home/wmind/anaconda3/share/jupyter/lab/staging/node_modules/enhanced-resolve/lib/ModuleKindPlugin.js:23:37
    at /home/wmind/anaconda3/share/jupyter/lab/staging/node_modules/enhanced-resolve/lib/Resolver.js:238:5
    at eval (eval at create (/home/wmind/anaconda3/share/jupyter/lab/staging/node_modules/tapable/lib/HookCodeFactory.js:33:10), <anonymous>:13:1)
    at /home/wmind/anaconda3/share/jupyter/lab/staging/node_modules/enhanced-resolve/lib/Resolver.js:238:5
    at eval (eval at create (/home/wmind/anaconda3/share/jupyter/lab/staging/node_modules/tapable/lib/HookCodeFactory.js:33:10), <anonymous>:13:1)
    at /home/wmind/anaconda3/share/jupyter/lab/staging/node_modules/enhanced-resolve/lib/forEachBail.js:30:14
    at /home/wmind/anaconda3/share/jupyter/lab/staging/node_modules/enhanced-resolve/lib/Resolver.js:238:5
    at eval (eval at create (/home/wmind/anaconda3/share/jupyter/lab/staging/node_modules/tapable/lib/HookCodeFactory.js:33:10), <anonymous>:13:1)
    at /home/wmind/anaconda3/share/jupyter/lab/staging/node_modules/enhanced-resolve/lib/UnsafeCachePlugin.js:37:5
    at /home/wmind/anaconda3/share/jupyter/lab/staging/node_modules/enhanced-resolve/lib/Resolver.js:238:5
    at eval (eval at create (/home/wmind/anaconda3/share/jupyter/lab/staging/node_modules/tapable/lib/HookCodeFactory.js:33:10), <anonymous>:13:1)
    at /home/wmind/anaconda3/share/jupyter/lab/staging/node_modules/enhanced-resolve/lib/Resolver.js:238:5
    at eval (eval at create (/home/wmind/anaconda3/share/jupyter/lab/staging/node_modules/tapable/lib/HookCodeFactory.js:33:10), <anonymous>:14:1)
    at /home/wmind/anaconda3/share/jupyter/lab/staging/node_modules/enhanced-resolve/lib/Resolver.js:238:5
    at eval (eval at create (/home/wmind/anaconda3/share/jupyter/lab/staging/node_modules/tapable/lib/HookCodeFactory.js:33:10), <anonymous>:14:1)
    at /home/wmind/anaconda3/share/jupyter/lab/staging/node_modules/enhanced-resolve/lib/Resolver.js:238:5
resolve 'worker-loader' in '/home/wmind/anaconda3/share/jupyter/lab/staging/node_modules/pdfjs-dist'
  Parsed request is a module
  using description file: /home/wmind/anaconda3/share/jupyter/lab/staging/node_modules/pdfjs-dist/package.json (relative path: .)
    resolve as module
      /home/wmind/anaconda3/share/jupyter/lab/staging/node_modules/pdfjs-dist/node_modules doesn't exist or is not a directory
      /home/wmind/anaconda3/share/jupyter/lab/staging/node_modules/node_modules doesn't exist or is not a directory
      /home/wmind/anaconda3/share/jupyter/lab/node_modules doesn't exist or is not a directory
      /home/wmind/anaconda3/share/jupyter/node_modules doesn't exist or is not a directory
      /home/wmind/anaconda3/share/node_modules doesn't exist or is not a directory
      /home/wmind/anaconda3/node_modules doesn't exist or is not a directory
      /home/node_modules doesn't exist or is not a directory
      /node_modules doesn't exist or is not a directory
      looking for modules in /home/wmind/node_modules
        No description file found
      looking for modules in /home/wmind/anaconda3/share/jupyter/lab/staging/node_modules
        using description file: /home/wmind/anaconda3/share/jupyter/lab/staging/package.json (relative path: ./node_modules)
          using description file: /home/wmind/anaconda3/share/jupyter/lab/staging/package.json (relative path: ./node_modules/worker-loader)
            no extension
              /home/wmind/anaconda3/share/jupyter/lab/staging/node_modules/worker-loader doesn't exist
            .js
              /home/wmind/anaconda3/share/jupyter/lab/staging/node_modules/worker-loader.js doesn't exist
        No description file found
            .json
              /home/wmind/anaconda3/share/jupyter/lab/staging/node_modules/worker-loader.json doesn't exist
        no extension
          /home/wmind/node_modules/worker-loader doesn't exist
            as directory
              /home/wmind/anaconda3/share/jupyter/lab/staging/node_modules/worker-loader doesn't exist
        .js
          /home/wmind/node_modules/worker-loader.js doesn't exist
        .json
          /home/wmind/node_modules/worker-loader.json doesn't exist
        as directory
          /home/wmind/node_modules/worker-loader doesn't exist
child_process.js:651
    throw err;
    ^

Error: Command failed: /home/wmind/anaconda3/share/jupyter/lab/staging/node_modules/.bin/webpack --config webpack.prod.minimize.config.js
    at checkExecSyncError (child_process.js:630:11)
    at Object.execFileSync (child_process.js:648:15)
    at Object.<anonymous> (/home/wmind/anaconda3/share/jupyter/lab/staging/node_modules/@jupyterlab/buildutils/lib/ensure-max-old-space.js:50:17)
    at Module._compile (internal/modules/cjs/loader.js:1138:30)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:1158:10)
    at Module.load (internal/modules/cjs/loader.js:986:32)
    at Function.Module._load (internal/modules/cjs/loader.js:879:14)
    at Function.executeUserEntryPoint [as runMain] (internal/modules/run_main.js:71:12)
    at internal/main/run_main_module.js:17:47 {
  status: 1,
  signal: null,
  output: [ null, null, null ],
  pid: 5336,
  stdout: null,
  stderr: null
}
error Command failed with exit code 1.
info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.

[LabBuildApp] JupyterLab failed to build
[LabBuildApp] Traceback (most recent call last):

[LabBuildApp]   File "/home/wmind/anaconda3/lib/python3.8/site-packages/jupyterlab/debuglog.py", line 47, in debug_logging
    yield

[LabBuildApp]   File "/home/wmind/anaconda3/lib/python3.8/site-packages/jupyterlab/labapp.py", line 138, in start
    raise e

[LabBuildApp]   File "/home/wmind/anaconda3/lib/python3.8/site-packages/jupyterlab/labapp.py", line 134, in start
    build(name=self.name, version=self.version,

[LabBuildApp]   File "/home/wmind/anaconda3/lib/python3.8/site-packages/jupyterlab/commands.py", line 459, in build
    return handler.build(name=name, version=version, static_url=static_url,

[LabBuildApp]   File "/home/wmind/anaconda3/lib/python3.8/site-packages/jupyterlab/commands.py", line 661, in build
    raise RuntimeError(msg)

[LabBuildApp] RuntimeError: JupyterLab failed to build

[LabBuildApp] Exiting application: JupyterLab
```

--- a/dev_mode/package.json
+++ b/dev_mode/package.json
@@ -84,6 +84,7 @@
     "webpack-cli": "^3.3.10",
     "webpack-merge": "^4.2.2",
     "whatwg-fetch": "^3.0.0",
+    "worker-loader": "^2.0.0",
     "yarn-deduplicate": "^1.1.1"
   },
   "resolutions": {

